lib/src/great_circle.dart

```diff
@@ -0,0 +1,103 @@
+import 'dart:math' as math;
+import 'package:turf/turf.dart';
+import 'helpers.dart';
+
+/// Calculates the great circle route between two points on a sphere 
+/// 
+/// Useful link: https://en.wikipedia.org/wiki/Great-circle_distance
+ 
+List<List<double>> greatCircle(
```

Comment:
please make sure our function returns something similar to the turfJS version.

Let's return a Feature with either 
a LineString or MultiLineString as the payload.

https://turfjs.org/docs/api/greatCircle